src/parser/parser.cc

```diff
@@ -417,7 +417,7 @@ class Parser {
    * Useful for matching optional tokens, effectively looksahead by one.
    */
   bool WhenMatch(const TokenType& token_type) {
-    DLOG(INFO) << "Parser::WhenMatch: Peek() == " << Peek();
+    // DLOG(INFO) << "Parser::WhenMatch: Peek() == " << Peek();
```

Comment:
Ah, right! I started on per-file VLOGs, but hosted that out of here. Commented these out since I couldn't see the wood for the trees, will undo sorry.